RATapi/utils/plotting.py

```diff
@@ -173,9 +203,37 @@ def plot_ref_sld(
     data.subRoughs = results.contrastParams.subRoughs
     data.resample = RATapi.inputs.make_resample(project)
 
+    if bayes:
+        try:
+            # the predictionIntervals data consists of 5 rows:
+            # row 0: min with 95% confidence
+            # row 1: min with 65% confidence
+            # row 2: mean
+            # row 3: max with 65% confidence
+            # row 4: max with 95% confidence
+            interval = [0, 4] if bayes == 95 else [1, 3]
```

Comment:
sure: was replicating the behaviour of `bayesShadedPlots` but of course that's a separate function to the unshaded plotting in that case

Reply:
well the current behaviour is that it reports 65% for any possible input except `95` OR `None` (default) - this line only runs if Bayes is actually given a value. should we warn/error the user if they haven't put in either of those options since it's either undefined or a mistake?

Reply:
I was thinking about this a bit more last night and yes, that's the right way to go.